Sources/TuistKit/Commands/GenerateCommand.swift

```diff
@@ -37,15 +38,16 @@ public struct GenerateCommand: AsyncParsableCommand, HasTrackableParameters {
 
     @Flag(
         name: .shortAndLong,
-        help: "Don't open the project after generating it."
+        help: "Don't open the project after generating it.",
+        envKey: .generateOpen
     )
-    var noOpen: Bool = false
+    var open: Bool = true
```

Comment:
This is a breaking change. I'd consider adopting [the flag inversion](https://swiftpackageindex.com/apple/swift-argument-parser/1.4.0/documentation/argumentparser/flaginversion) option supported by argument parser, and have `TUIST_GENERATE_OPEN=1/0` (truthy == open / falsy == no open)

This comment applies to all the breaking changes in commands.

Reply:
Well thought of 👏🏼 . I completely missed that in the changes.